docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
+#### How to unjail
 
-**where *x* is in years and *f(x)* is in millions of CUDOS.**
+To unjail, the validator must send a transaction from the jailed validator account via the `Cudos-noded CLI`. 
 
-* Hence the amount to be distributed in a given time interval is obtained by integrating the above function over that interval.
+```shell
+cudos-noded tx slashing unjail --from mykey [flags]
+```
 
-* Due to the assumption that tx fees will eventually be the main source of revenue, staking rewards start high and then decrease quadratically in time.
+### Unbonding period
 
-:::tip Important
+If a validator is jailed or leaves the bonded validators set, it enters an **unbonding period**. A validator that has failed to get itself back into the bonded validators set *within 21 days* is said to be **unbonded**.
 
-Important note: Validators need to use the --commission flag in the CLI in order to withdraw their commission.
+:::caution 
+Slashing can still occur during the unbonding period.
 :::
 
-## Jailing/Slashing
+### Token holder staking
 
-* Tokens delegated to a bonded validator is slashed if there is evidence of the validator misbehaving.
+Anyone holding CUDOS tokens can delegate or stake their tokens to one or more **validator nodes** of their choice. Validators and Stakers then share rewards **based on the amounts they have contributed to the pool**.
 
-* This means that when a validator’s tokens are slashed by a certain percentage, so are it delegators’.
+Validator Operators can set their own Commission to charge Stakers.**Commission rates** can change *once every day* however, the **Maximum Commission Rate** cannot be exceeded. Commission fees can be viewed under [Validator details in the Cudos Explorer](https://explorer.cudos.org/validators). 
 
-* If a validator fails to sign 90% of the blocks within a 19200-block interval (i.e. 17280 blocks), then it is jailed:
+![validator-details](@site/static/img/validator-details.png)
 
-    * It becomes not-bonded and begins unbonding
-    * It stops signing and proposing blocks
-    * It stops earning block rewards
-    * It’s stake gets slashed by 0.01%
+## Redelegation of tokens
 
-* The validator needs to unjail itself after at least 600 seconds of being jailed. There is an unjail transaction that can be sent via the CLI from within the Validator account/node. 
-
-```shell
-cudos-noded tx slashing unjail --from mykey [flags]
-```
+A **staker/delegator** can choose to **undelegate** their tokens at any time. 
 
+A **staker/delegator** can **redelegate** their tokens from **Validator A** to **Validator B** without any penalties or extra waiting times. However, they need to wait for 21 days if they want to further redelegate any amount of tokens from Validator B.
 
-For each validator, jailing starts after signed_blocks_window + start_height many blocks.
-We have signed_blocks_window = 19200.
+To redelegate tokens from **Validator B** to **Validator C** there is a *21 day waiting period*.
```

Comment:
Isn't this sentence the same as the previous one?

Reply:
Ah to clarify
1. I have delegated my tokens from Validator A to Validator B 
Scenario 1:  I now wish to delegate more tokens to Validator B. - Does this require a 21 day waiting period? Or is this only if it is redelegating the tokens from Validator A again. What if I am adding extra tokens from a new account? 
Scenario 2:  I have now decided that I was wrong to redelegate to Validator B, I now want to redelegate tokens from Validator B to Validator C. - Does this require a 21 day waiting period?

Reply:
Hey @ilker-colak would you mind clarifying these scenarios for me. Thanks.

Reply:
After redelegating from Validator A to Validator B, the 21-day waiting period applies only if you want to redelegate **from** Validator B to another validator. I don't remember the exact name, but this period is called "the redelagation maturing period" or something like that. Basically you can redelagate your already redelagated tokens only after they mature. And since your already redelegated tokens are mixed with your other tokens on Validator B, the system cannot distinguish them and locks them all.

Let's say you have 10 tokens on Validator A, and 100 tokens on Validator B. Then you redelagte 1 token from Validator A to Validator B. Now you have to wait for 21 days before you can redelagate any of your 101 tokens on Validator B to any other validator.

Scenario 1: You can redelagate to Validator B from any validator, or directly delegate to it from any account, with no restrictions.

Scenario 2: No waiting period.

Scenarios 3: 21-day waiting period applies.

Reply:
done